src/coreclr/tools/cdac-build-tool/data-descriptor-blob.md

```diff
@@ -108,20 +116,23 @@ struct BinaryBlobDataDescriptor
         uint32_t GlobalLiteralValuesStart;
 
         uint32_t GlobalPointersStart;
+        uint32_t GlobalStringValuesStart;
```

Comment:
If we're adding new members to the directory, shouldn't we add them all at the end? Or is the goal for any old code based on the old directory layout to break in a loud and confusing way that makes it more obvious that it needs to be updated? I'd personally love to see a version at the head of the directory or somewhere else appropriate if we're going to insert data in the middle of it instead of at the end.

Reply:
That sounds fine then. I'd love to see a basic version header that we can use to detect cdac-build-tool being out of date but I don't think it needs to block this PR, it can be something we do the first time we have that problem crop up anywhere.